Source/Charts/Data/Implementations/Standard/ChartDataEntry.swift

```diff
@@ -108,32 +99,18 @@ open class ChartDataEntry: ChartDataEntryBase
     }
 }
 
-public func ==(lhs: ChartDataEntry, rhs: ChartDataEntry) -> Bool
-{
-    if lhs === rhs
-    {
-        return true
-    }
-    
-    if !lhs.isKind(of: type(of: rhs))
+// MARK: Equatable
+extension ChartDataEntry/*: Equatable*/ {
```

Comment:
Why is equatable commented out?

Reply:
Because `NSObject` adopts `Equatable` and you can't declare conformance twice. When Obj-C support is dropped, then we will need to uncomment it.